CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/plugins/CopyCustomMarkersToGraphPlugin.java

```diff
@@ -49,7 +48,7 @@
  * @author cygnus_x-1
  */
 @ServiceProvider(service = Plugin.class)
-@PluginInfo(pluginType = PluginType.COPY, tags = {"COPY"})
+@PluginInfo(tags = {"MODIFY"})
 @NbBundle.Messages("CopyCustomMarkersToGraphPlugin=Copy Custom Markers to Graph")
 public class CopyCustomMarkersToGraphPlugin extends SimpleEditPlugin {
```

Comment:
You've gone with MODIFY which could be fine but in my head I could also make an argument for IMPORT (from map) or CREATE. Thoughts @arcturus2?

Reply:
I can see that method, happy with either from my end.

Reply:
Hi @antares1470 @aldebaran30701! The input to this seems to be the custom marks from the Map View which is still internal to Constellation so I would keep "MODIFY". If it was a file import then that's where I'd prefer using IMPORT.